jobs/github/mergejobs.yaml

```diff
@@ -200,92 +200,92 @@
 
 ############################################################# PYLIBJUJU TESTS ##
```

Comment:
This header should probably be removed as well right?

Reply:
I will leave it there, just in case we regret about removing the python-libjuju testing here.